docs/guides/variables.md

```diff
@@ -54,29 +54,32 @@ In addition to the UI and API, variables can be referenced in code and in certai
 
 ### In Python code
 
-You can access any variable via the Python SDK via the `.get()` method. If you attempt to reference a variable that does not exist, the method will return `None`.
+You can access any variable via the Python SDK via the `Variable.get()` method. If you attempt to reference a variable that does not exist, the method will return `None`. You can create variables via the Python SDK with the `Variable.set()` method. Note that if a variable of the same name exists, you'll need to pass `overwrite=True`.
 
 ```python
 from prefect import variables
 
-# from a synchronous context
-answer = variables.get('the_answer')
-print(answer)
+# setting the variable
+variable = variables.Variable.set(name="the_answer", value="42")
```

Comment:
2 tiny things here

1. can we update the examples to look like:
```python
from prefect.variables import Variable

# set a variable
variables.Variable.set(name="the_answer", value="42")
...
```
I think it's clearer and less verbose

2. can we also add an example line for `overwrite=True`? I am guilty of jumping straight to the code examples and skipping the words.